.circleci/config.yml

```diff
@@ -0,0 +1,113 @@
+# These environment variables must be set in CircleCI UI
+#
+# DOCKERHUB_REPO - docker hub repo, format: <username>/<repo>
+# DOCKER_USER    - login info for docker hub
+# DOCKER_PASS
+#
+version: 2.1
+commands:
+  write-version:
```

Comment:
Yes, please add this to the version endpoint!

Reply:
Right, but since this is a set of scripts that are used to run testing against the contile service, there's no `/__version__` endpoint to speak of. These should not be set up as a normal service (with the other dockerflow endpoints), but instead be used as a base for molotov or similar test runs.